configs/template_config.yaml

```diff
@@ -0,0 +1,94 @@
+# USAGE: The YAML configuration contains all necessary paths and parameters to download data
+#        train a tokenizer, tokenize the data, train a model, and evaluate the model.
+#
+#        To prepare a YAML config for usage, create a copy of template_config.yaml 
+#        in the user_configs folder and fill in the necessary parameters.
+#        Path names need to be specified properly. These should be absolute paths.
+#        A suggestion for path structure is given in the template.
+#        The YAML config file is then passed as an argument to any scripts.
+#
+#        For example, to train a model, run:
+#        python train_model.py ./configs/my_config.yaml
```

Comment:
I am not certain, so correct me if I am wrong, but would it be a good idea to suggest the following instead?:

python train_model.py ./configs/user_configs/my_config.yaml.

If I understand the new gitignore file, that folder is where everyone should put their YAML files.

Reply:
Good catch